source/rst-examples.rst

```diff
@@ -15,21 +28,89 @@ Jonathan H. Wage wrote these examples!
 
         echo 'Hello World';
 
-.. versionadded:: 2.2
+Code Examples
+-------------
 
-This was added new in version 2.2.
+Code
+~~~~
 
-.. code-block:: php
+.. code-block:: rst
```

Comment:
`rst` currently doesn't do anything because it isn't a valid code-highlighting on the Doctrine RST rendering. This can be introduced to the rendering later though.